gen3users/conversion.py

```diff
@@ -24,13 +25,6 @@
 )
 
 
-# "base_abac.json" contains ABAC elements that are included in all
-# user.yaml files
```

Comment:
maybe we could move this comment to `base_abac.py`

Reply:
yeah, I can do that. I should stop making PRs before things are actually ready 😓 Still testing, I'm not sure this really fixes things